tests/e2e/AreaChart/AreaChart.spec.ts

```diff
@@ -7,15 +7,17 @@ describe("Area chart", () => {
     });
 
     it("should generate a chart", () => {
-        chart.svgElement.waitForVisible();
+        chart.svgElement.waitForDisplayed();
         const isExist = chart.svgElement.isExisting();
 
         expect(isExist).toBeTruthy();
     });
 
     it("should be generated with two traces", () => {
-        chart.traces.waitForValue();
+        browser.waitUntil(() => {
+            return chart.traces.map((elem) => elem.isDisplayed()).length > 1;
```

Comment:
I think you need to filter here.

Reply:
I didn't understand the suggestion of use filter here. What would be the difference?